ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/sflow.schema.yml

```
@@ -13,6 +13,8 @@ keys:
         - str
       dangerous:
         type: bool
+      polling_interval:
+        type: int
```

Apologies for not catching this earlier. Please add this and then I will merge.
```suggestion
        type: int
        convert_types:
        - str
        description: Polling interval in seconds
```

---

ahh, this will convert automatically to INT if the user enters a string?

---

Yes we have builtin some auto-type-conversion to "forgive" the user, since YAML or Ansible with inline Jinja can be difficult sometimes.